### ISSUE TITLE
Steering DP definitions in the right direction

### ISSUE THREAD

**Issue by cjpatton** [wangshan/draft-wang-ppm-differential-privacy#26]
While reviewing the binary randomized response mechanism (#22) for which we have a sort of "one-off" DP definition, I had the following thought.

In the medium term, I think we should spend time thinking through how this notion of DP relates to the notion of DP we've already defined.

Long term, I think our goal for PPM should be to have a single (or as few as possible) notions of DP. That way we have a standard methodology (or methodologies) for evaluating DP mechanisms and policies. The fact that there are so many DP notions in the literature that may or may not be comparable is a huge problem for standardization.

Relatedly, I believe there is an important research question here: for a given application (DAP) systematize the DP notions in the literature that apply such that we can compare them to one another (at least qualitatively, but ideally quantitatively). Presumably there are trade-offs to be made, depending on which notion of DP you pick; what we want at the end of the day is the right DP notion that allows us to make sense of what to do for DAP.

_Originally posted by @cjpatton in https://github.com/wangshan/draft-wang-ppm-differential-privacy/pull/22#discussion_r1365830045_

**Comment by junyechen1996** [comment]
I agree we shouldn't complicate the spec if we don't have to. The case with binary randomized response is that it is a simple local randomizer, but only works in deletion-DP model, which is worth mentioning because a `EPSILON_0`-DP in deletion-DP translates to an upper bound of `2 * EPSILON_0`-DP in replacement-DP (Definition II.4 in [EFMR+20](https://arxiv.org/pdf/2001.03618.pdf) and Definition C.1 in [MJTB+22](https://arxiv.org/pdf/2211.10082.pdf)). I think we only need to deal with deletion-DP in the Client-DP mechanisms, but from an aggregate result point of view, we always want to provide DP in replacement-DP model, which is how DP is defined in overview section of the draft. Hence currently we only introduce deletion-DP in binary randomized response.

There are other notions of DP we may need to introduce, e.g. other definitions of DP budget instead of pure `EPSILON`-DP and `(EPSILON, DELTA)`-DP, e.g. [Rényi-DP](https://arxiv.org/abs/1702.07476), [Concentrated-DP](https://arxiv.org/abs/1605.02065), which are more optimal during composition of DP budgets if we need to enforce DP budgets. But we can think about these when we get there.

> for a given application (DAP) systematize the DP notions in the literature that apply such that we can compare them to one another (at least qualitatively, but ideally quantitatively).

I agree we should compare the tradeoffs, so users can pick the right DP policy based on the deployment environment they are in. I assume that's one of the goals of this draft. To start with, quantitatively, we can probably compare utility, communication cost, and qualitatively, trust model?
